lib/units/auth/openid.js

```diff
@@ -0,0 +1,73 @@
+var http = require('http')
+var url = require('url');
+var querystring = require('querystring');
+
+var openid = require('openid');
+var express = require('express')
+var urljoin = require('url-join')
+
+var logger = require('../../util/logger')
+var jwtutil = require('../../util/jwtutil')
+var urlutil = require('../../util/urlutil')
+
+module.exports = function(options) {
+  var extensions = [new openid.SimpleRegistration({
+    "email" : true,
+    "fullname" : true,
+  })];
+  var relyingParty = new openid.RelyingParty(
+    urljoin(options.appUrl, "/auth/verify"),
```

Comment:
Very minute but, it will be better if you can make this end point as `/auth/verify` -> `/auth/openid/verfity`